### PR TITLE
cleos remove wasm to wast conversion 📦

### PR DESCRIPTION
**Pull request by heifner**
## Change Description

- Update `cleos` to not perform wasm to wast conversion.
- Users can use standard WebAssembly tooling to convert from wasm to wast if needed.
- Removal announcement: https://github.com/EOSIO/eos/issues/7597

## Change Type
**Select *ONE*:**
- [ ] Documentation

- [ ] Stability bug fix

- [x] Other

- [ ] Other - special case

## Testing Changes
**Select *ANY* that apply:**
- [ ] New Tests

- [ ] Existing Tests

- [ ] Test Framework

- [ ] CI System

- [ ] Other

## Consensus Changes
- [ ] Consensus Changes

## API Changes
- [ ] API Changes

## Documentation Additions
- [x] Documentation Additions

### CODE REVIEW

File: libraries/chain/include/eosio/chain/wast_to_wasm.hpp [hunk 1]
```diff
@@ -6,6 +6,6 @@ namespace eosio { namespace chain {
 
 std::vector<uint8_t> wast_to_wasm( const std::string& wast );
 std::string  wasm_to_wast( const std::vector<uint8_t>& wasm, bool strip_names );
-std::string  wasm_to_wast( const uint8_t* data, uint64_t size, bool strip_names );
+std::string  wasm_to_wast( const uint8_t* data, uint64_t size, bool strip_names, bool check_limits = true );
```
Comment (spoonincode) [inline libraries/chain/include/eosio/chain/wast_to_wasm.hpp]: can all the wasm_to_wast stuff be ripped out?

Comment (heifner) [reply libraries/chain/include/eosio/chain/wast_to_wasm.hpp]: Yes. It is not used, so might as well.

Comment (heifner) [reply libraries/chain/include/eosio/chain/wast_to_wasm.hpp]: Removed.

Comment (spoonincode) [reply libraries/chain/include/eosio/chain/wast_to_wasm.hpp]: it's probably possible to purge more, down in the libraries/wasm-jit/Source/WAST (and include dir). That would be highly desirable, but maybe not critical at the moment

**Review by spoonincode — APPROVED** [review]

File: libraries/chain/include/eosio/chain/wast_to_wasm.hpp (hunk 1 above)
Comment (spoonincode) [reply libraries/chain/include/eosio/chain/wast_to_wasm.hpp]: ah, hm, can't remove all too much in there since unit tests since use wast

**Pull request merged**